app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @first_user = User.first
+  before_action :authenticate_user!
+  before_action :authenticate_user!
```

Comment:
- [ ] Great work adding authentication before loading the page, however, you have repeated this line of code twice, which is not a good practise.